inst/compareScenarios/csEDGET_main.Rmd

```diff
@@ -17,15 +17,15 @@ output:
     number_sections: yes
 geometry: "a4paper,landscape,left = 0.5cm,right = 0.5cm,top = 0.5cm,bottom = 0.5cm,footnotesep=0.0cm,footskip = 0.1cm"
 params:
-  mifScen: ""
+  mifScen: !r c()
```

Comment:
Does this work now differently? I remember that the compScen was super picky about the formatting.